src/consent-selections.ts

```diff
@@ -16,10 +16,14 @@ export function getConsentSelections(airgap: AirgapAPI): ConsentSelection {
   const initialConsentSelections: ConsentSelection = {};
 
   // Get the purposes for processing configured for this organization.
+  const regimePurposes = airgap.getRegimePurposes();
   const purposeTypes = airgap.getPurposeTypes();
+  const applicablePurposes = Object.keys(purposeTypes).filter((purpose) =>
```

Comment:
Seems redundant to filter applicable purposes out of all purposes when you could just use the applicable purposes as-is

Reply:
Lol true